src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -387,6 +387,14 @@ public static DataMarshaler OutputMarshaler
                             {
                                 return outputMarshaler.Marshal(clrObj);
                             }
+
+                            if (IsMarkedToSkipConversion(pyObj))
+                            {
+                                var globalScope = PyScopeManager.Global.Get(globalScopeName);
+                                globalScope.Set(pyObj.Handle.ToString(), pyObj);
+                                return new DynamoCPythonHandle(pyObj.Handle);
```

Comment:
Minor but these 3 lines of code are repeated in 2 places - here and further down below. Might be worth refactoring into a helper method?

Reply:
wil fix.